src/python/CRABClient/ClientUtilities.py

```diff
@@ -619,8 +619,14 @@ def setSubmitParserOptions(parser):
                            dest = 'dryrun',
                            default = False,
                            action = 'store_true',
-                           help = "Do not actually submit the task; instead, return how many jobs this task would create, along with processing time estimates.")
+                           help = "Do not actually submit the task; instead, return how many jobs this task would create, "\
+                                  "along with processing time and memory consumption estimates.")
 
+    parser.add_option('--skip-estimates',
```

Comment:
@annawoodard , can you document this in the twiki for the commands (I'll let @AndresTanasijczuk tell you which is the right URL :)).